### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by appinteractive**
This pull request was initiated by Lokalise (user Mogge) at 2019-11-18 21:33:48

### CODE REVIEW

**Comment by Mogge** [comment]
I did this mainly for testing. Is the path correct now?

**Comment by roschaefer** [comment]
@Mogge yes this is a lot better.

File: webapp/locales/ru_RU.json [hunk 1]
```diff
@@ -0,0 +1,2246 @@
+{
```
Comment (roschaefer) [inline webapp/locales/ru_RU.json]: @Mogge this will have no effect. The file name has the extension `_RU` to it and our software won't recognize it.

Comment (Mogge) [reply webapp/locales/ru_RU.json]: OK. I try to change the name of the language in lokalise.

Comment (roschaefer) [reply webapp/locales/ru_RU.json]: @Mogge can you remove this file from the commit? So we can merge both this PR and https://github.com/Human-Connection/Human-Connection/pull/2277?

Comment (roschaefer) [reply webapp/locales/ru_RU.json]: I'll approve then

Comment (Mogge) [reply webapp/locales/ru_RU.json]: Done

File: webapp/locales/de.json [hunk 2]
```diff
@@ -1,736 +1,2246 @@
...
-  "components": {
-    "password-reset": {
-      "request": {
-        "form": {
-          "description": "Eine Mail zum Zurücksetzen des Passworts wird an die angegebene E-Mail Adresse geschickt.",
-          "submit": "Email anfordern",
-          "submitted": "Eine E-Mail mit weiteren Instruktionen wurde verschickt an <b>{email}</b>"
+    "login": {
+        "copy": {
+            "translation": "Falls Du bereits ein Konto bei Human Connection hast, melde Dich bitte hier an.",
```
Comment (roschaefer) [inline webapp/locales/de.json]: Why do we have those keys "translation" and "notes" all over the place? @Mogge misconfiguration?

Comment (Mogge) [reply webapp/locales/de.json]: See #2305. I changed the export to
grafik

I close this PR

**Comment by Mogge** [comment]
Closed due to obsolete

**Pull request closed without merging**